app/src/main/java/org/simple/clinic/newentry/form/DateOfBirthField.kt

```diff
@@ -4,20 +4,18 @@ import org.simple.clinic.newentry.form.ValidationError.DateIsInFuture
 import org.simple.clinic.newentry.form.ValidationError.InvalidDateFormat
 import org.simple.clinic.newentry.form.ValidationError.MissingValue
 import java.time.LocalDate
-import java.time.format.DateTimeFormatter
 import java.time.format.DateTimeParseException
 
 data class DateOfBirthField(
-    private val dateTimeFormatter: DateTimeFormatter,
+    private val parsedDate: (String) -> LocalDate,
```

Comment:
I initially went with `dateParser`, but it felt accurate to use this, since we are using a return value from that function. WDYT? should I change it?

Reply:
We could also just call it `parseDate` since that indicates a verb.